lgc/patch/Gfx6ConfigBuilder.cpp

```diff
@@ -934,6 +934,13 @@ void ConfigBuilder::buildPsRegConfig(ShaderStage shaderStage, T *pConfig) {
     pointCoordLoc = resUsage->inOutUsage.builtInInputLocMap[BuiltInPointCoord];
   }
 
+  unsigned viewportIndexLoc = InvalidValue;
+  if (resUsage->inOutUsage.builtInInputLocMap.find(BuiltInViewportIndex) !=
```

Comment:
Can save the iterator from `.find()` to a local variable so that you don't have to perform a second lookup inside the if?

Reply:
Done. Also change the previous if to match the behavior.